modules/installation-configuration-parameters.adoc

```diff
@@ -2736,107 +2730,73 @@ The list of datacenters must match the list of datacenters specified in the `vce
+endif::vsphere[]
+ifdef::agent[]
 |The path to the vSphere datastore that holds virtual machine files, templates, and ISO images.
-[IMPORTANT]
-====
-You can specify the path of any datastore that exists in a datastore cluster.
+
+*Important:* You can specify the path of any datastore that exists in a datastore cluster.
 By default, Storage vMotion is automatically enabled for a datastore cluster.
 Red{nbsp}Hat does not support Storage vMotion, so you must disable Storage vMotion to avoid data loss issues for your {product-title} cluster.
 
-If you must specify VMs across multiple datastores, use a `datastore` object to specify a failure domain in your cluster's `install-config.yaml` configuration file.
-For more information, see "VMware vSphere region and zone enablement".
-====
-|String
-
-|platform:
-  vsphere:
-    failureDomains:
-      topology:
-        resourcePool:
-// When this content is added to vSphere docs, the line below should likely say "where the installation program creates the virtual machines".
-|Optional: The absolute path of an existing resource pool where the user creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`.
-// Commenting out the line below because it doesn't apply to Agent, but it may be needed when this content is brought into the regular vSphere docs.
-// If you do not specify a value, resources are installed in the root of the cluster, for example `/example_datacenter/host/example_cluster/Resources`.
+If you must specify VMs across multiple datastores, use a `datastore` object to specify a failure domain in your cluster's `install-config.yaml` configuration file. For more information, see "VMware vSphere region and zone enablement".
+endif::agent[]
 |String
 
 |platform:
   vsphere:
     failureDomains:
       topology:
         folder:
-// When this content is added to vSphere docs, the line below should likely say "where the installation program creates the virtual machines", and should be Optional.
-|The absolute path of an existing folder where the user creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`.
-// Commenting out the two lines below because they don't apply to Agent, but they may be needed when this content is brought into the regular vSphere docs.
-// If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID.
-// If you are providing the infrastructure for the cluster and you do not want to use the default `StorageClass` object, named `thin`, you can omit the `folder` parameter from the `install-config.yaml` file.
+|Optional parameter. The absolute path of an existing folder where the user creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`.
```

Comment:
Is _Optional parameter._ proper usage? I don't see it after a quick search of the docs.

Reply:
![Screenshot from 2024-04-22 14-52-39](https://github.com/openshift/openshift-docs/assets/57954076/3e4e2439-40af-4f3e-9204-7f3035f881bf)

Reply:
@bscott-rh . Are you OK with this format: "Optinal:" versus "Optional parameter"?

Reply:
Yes, "Optional:" is fine